create/tests/force_clean_env.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
```

Comment:
Once we have this https://github.com/comit-network/create-comit-app/issues/388 we can remove this file again.

Reply:
Can you please explain more why this fail is necessary?

Reply:
Our environment does not terminate deterministically which will lead to unexpected errors during e2e test execution. If we only run 1 test, then this is obviously not an issue, but with 2 the second might randomly fail. 

See here: 
* https://github.com/comit-network/create-comit-app/runs/400255524
* https://github.com/comit-network/create-comit-app/runs/400238837